source/_integrations/smtp.markdown

```diff
@@ -21,9 +21,13 @@ To enable notification by e-mail in your installation, add the following to your
 notify:
   - name: "NOTIFIER_NAME"
     platform: smtp
+    server: "smtp.example.com"
+    username: "YOUR_USER_NAME"
+    password: "YOUR_PASSWORD"
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample.

Reply:
I was divided in this one, on one hand, we have the pattern of not including optional configurations in the configuration example, and on the other, we have that the most common use case for SMTP integrations is to use an external SMTP provider that will usually require a username and password.

Given that we are including provider-specific configuration in the end I will not strongly argue about keeping those optional Configuration variables, so I am good about removing them if I could not convince you @frenck :)

Reply:
There is no discussion on this part, this is following our documentation standards. Please remove those optional parameters. Thanks 👍

Reply:
Consider it done! I will revert to the original example, below, as the server is also an optional variable.
`# Example configuration.yaml entry
notify:
  - name: "NOTIFIER_NAME"
    platform: smtp
    sender: "YOUR_SENDER"
    recipient: "YOUR_RECIPIENT"`